Tests/UnitTests/Networking/Backend/__Snapshots__/BackendPostReceiptDataTests/iOS12-testDoesNotPostConsentStatus.1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "headers" : {
+    "Authorization" : "Bearer asharedsecret"
+  },
+  "request" : {
+    "body" : {
```

Comment:
@aboedo this is the "test". Equivalent to the normal one: https://github.com/RevenueCat/purchases-ios/blob/main/Tests/UnitTests/Networking/Backend/__Snapshots__/BackendPostReceiptDataTests/iOS16-testPostsReceiptDataCorrectly.1.json#L9